lib/get-unified-config.js

```diff
@@ -221,13 +222,30 @@ function getUnifiedConfig () {
     }
   })
 
+  //
+  // if an explicit trusted path has been provided, read the contents of the certificate
+  //
+  if ('trustedPath' in results.global) {
+    try {
+      const contents = fs.readFileSync(results.global.trustedPath, 'utf8')
+      results.global.certificates = contents
+    } catch (e) {
+      errors.push(['Cannot read certificate file', e])
+    }
+  }
+
   //
   // if the metricFormat hasn't been explicitely provided, infer it from the collector endpoint
+  // if no explicit trusted path has been provided also use the appoptics collector certificate
   //
   if (!('metricFormat' in results.global)) {
     if ('endpoint' in results.global && results.global.endpoint.includes('appoptics.com')) {
       // if a custom endpoint is provided and is an appoptics collector, only send TransactionResponseTime
       results.global.metricFormat = 1
+
+      if (!('certificates' in results.global)) {
+        results.global.certificates = require('./appoptics.crt')
+      }
```

Comment:
Minor, I know we shouldn't expect `metricsFormat` to be set since it's not advertised... but for the edge case (perhaps testing) where it is set, and we're pointing to the AO collector, seems this defaulting of certificate would be missed?  At the cost of duplicating that endpoint check I think it would be clearer if the certificate and metrics handling are separated.

Reply:
Good point, I separated the logic entirely